cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/SQLGenerator.scala

```diff
@@ -0,0 +1,654 @@
+      case q: DropView => parseDrop(q, schemaMap)
+      case q: DeleteRecord => parseDelete(q, schemaMap)
+      case _ => ???
+    }
+    s"$result"
+  }
+
+  def splitSchemaMap(schemaMap: Map[String, AbstractSchema]): (Map[String, Schema], Map[String, LookupSchema]) = {
+    val temporalSchemaMap = scala.collection.mutable.Map[String, Schema]()
+    val lookupSchemaMap = scala.collection.mutable.Map[String, LookupSchema]()
+
+    schemaMap.filter{ case(name, schema) =>
+      schema.isInstanceOf[Schema]
+    }.foreach{ case(name, schema) =>
+      temporalSchemaMap.put(name, schema.asInstanceOf[Schema])
+    }
+
+    schemaMap.filter{ case(name, schema) =>
+      schema.isInstanceOf[LookupSchema]
+    }.foreach{ case(name, schema) =>
+      lookupSchemaMap.put(name, schema.asInstanceOf[LookupSchema])
+    }
+
+    (temporalSchemaMap.toMap, lookupSchemaMap.toMap)
+  }
+
+  def parseCreate(create: CreateView, schemaMap: Map[String, AbstractSchema]): String = {
+    val (temporalSchemaMap, lookupSchemaMap) = splitSchemaMap(schemaMap)
+    val sourceSchema = temporalSchemaMap(create.query.dataset)
+    val resultSchema = calcResultSchema(create.query, sourceSchema)
+    val ddl: String = genDDL(create.dataset, sourceSchema)
+    val insert =
+      s"""
+     |replace into $quote${create.dataset}$quote ()
+     |(
+     |${parseQuery(create.query, schemaMap)}
+     |)
+      """.stripMargin.replace("T", " ").replace("Z", "")
+    ddl + insert
+  }
+
+  protected def genDDL(name: String, schema: Schema): String = {
+    def mkNestDDL(names: List[String], typeStr: String): String = {
+      names match {
+        case List(e) => s"  $e $typeStr"
+      }
+    }
+
+    val fields = schema.fieldMap.values.filter(f => f.dataType != DataType.Hierarchy && f != AllField).map {
+      f => mkNestDDL(f.name.split("\\.").toList, fieldType2ADMType(f) + (if (f.isOptional) " default null" else " not null"))
+    }
+    s"""
+       |create table if not exists $quote${name}$quote (
+       |${fields.mkString(",\n")}, primary key (${schema.primaryKey.map(key => key.name).mkString(",")})
+       |);
+      """.stripMargin
+  }
+
+  protected def fieldType2ADMType(field: Field): String = {
+    field.dataType match {
+      case DataType.Number => "double"
+      case DataType.Time => "datetime"
+      case DataType.Point => "point"
+      case DataType.Boolean => "tinyint"
+      case DataType.String => "varchar(255)"
+      case DataType.Text => "text"
+      case DataType.Json => "json"
+      case DataType.Bag => s"{{${fieldType2ADMType(Field("", field.asInstanceOf[BagField].innerType))}}}"
+      case DataType.Hierarchy => ???
+      case DataType.Record => ???
+    }
+  }
+
+  def parseAppend(append: AppendView, schemaMap: Map[String, AbstractSchema]): String = {
+    s"""
+       |replace into $quote${append.dataset}$quote (
+       |${parseQuery(append.query, schemaMap)}
+       |)""".stripMargin.replace("T", " ").replace("Z", "")
```

Comment:
The Date type's format only differs when inserting records: 
"yyyy-MM-dd'T'HH:mm:ss.SSS'Z'" vs "yyyy-MM-dd HH:mm:ss.SSS"